train_management/templates/train_management/traintimetabletemplate_list.html

```diff
@@ -79,22 +84,39 @@
             {% translate "template.train_timetable_template.add" %}
           </a>
           {% endif %}
-          <button id="daySelectionBtn" class="btn btn-primary btn-sm disabled">tmpGetSelections</button>
+          {% if train|length %}
+          <button id="saveTimetable" class="btn btn-primary btn-sm">Aus Vorlage erstellen</button>
+          {% endif %}
         </div>
       </div>
     </div>
   </div>
 </div>
+<form id="timetableForm" method="post" action="{% url 'train-timetable-template-use' %}">
+  {% csrf_token %}
+  <input type="hidden" id="timetableInput" name="timetables">
+  <input type="hidden" id="pk" name="pk" value="{{train}}">
+</form>
+
 
+{% endblock content %}
+{% block scripts %}
 <script>
-  var $table = $('#templateListTable');
-  var $button = $('#daySelectionBtn');
+  const saveTimetableButton = document.getElementById('saveTimetable');
+  const timetableForm = document.getElementById('timetableForm');
+  const formInput = timetableForm.querySelector('#timetableInput');
 
-  $(function() {
-      $button.click(function () {
-          alert('getSelections: ' + JSON.stringify($table.bootstrapTable('getSelections')))
-        })
+  function getIdSelections() {
+    return $.map($("#templateListTable").bootstrapTable('getSelections'), function (row) {
+      return row.id
     })
+  }
+
+  function saveTimetable() {
+        formInput.value = getIdSelections();
+        timetableForm.submit();
+    }
+    saveTimetableButton.addEventListener('click', saveTimetable);
```

Comment:
The jQuery way of doing that would rather be something like:
```
$(element).on('click', function () { /* do stuff */ });
```
Also checkout [this Stackoverflow Post](https://stackoverflow.com/a/6348597) to read up on the topic.